tests/src/TestHelpers.hpp

```diff
@@ -42,3 +46,21 @@ class ReceivedMessage
         return &this->t;
     }
 };
+
+inline std::ostream &operator<<(std::ostream &os, const QStringView &str)
+{
+    os << qUtf8Printable(str.toString());
+    return os;
+}
+
+inline std::ostream &operator<<(std::ostream &os, const QByteArray &bytes)
+{
+    os << qUtf8Printable(QString::fromUtf8(bytes));
```

Comment:
This is doing UTF8 → UTF16 → UTF8. There's [toStdString](https://doc.qt.io/qt-6/qbytearray.html#toStdString) and even this isn't _really_ needed.

Reply:
fixed in [`208b90a` (#5346)](https://github.com/Chatterino/chatterino2/pull/5346/commits/208b90a9b04a69162072ab0c366dbcac45bb8ef4)